src/main/resources/assets/sodium/lang/ru_ru.json

```
@@ -0,0 +1,66 @@
+  "sodium.options.cpu_render_ahead_limit.tooltip": "Задаёт максимальное количество кадров, которые ЦП может подготавливать для рендеринга на ГП. Очень низкие или высокие значения могут дестабилизировать частоту кадров.",
+  "sodium.options.cpu_render_ahead_limit.value": "%s кадр(ов)",
+  "sodium.options.allow_direct_memory_access.name": "Разрешить прямой доступ к памяти",
+  "sodium.options.allow_direct_memory_access.tooltip": "Если включено, некоторым критическим путям кода будет разрешено использовать прямой доступ к памяти для повышения производительности. Зачастую это сильно снижает перегрузку ЦП при рендеринге чанков и сущностей, но может усложнить диагностику некоторых ошибок и вылетов. Отключать это стоит только если вы следуете инструкциям или точно знаете, что делаете.",
+  "sodium.options.enable_memory_tracing.name": "Включить трассировку памяти",
+  "sodium.options.enable_memory_tracing.tooltip": "Функция отладки. Если включено, трассировка стека будет собираться вместе с распределением памяти, чтобы помочь улучшить диагностическую информацию при обнаружении утечек памяти.",
+  "sodium.options.performance_impact_string": "Влияние на производительность: %s",
+  "sodium.options.use_persistent_mapping.name": "Использовать постоянный буфер",
+  "sodium.options.use_persistent_mapping.tooltip": "Если включено, небольшое количество памяти будет постоянно использоваться в качестве промежуточного буфера для загрузки чанков, помогая уменьшить перегрузку ЦП и нестабильность времени отрисовки кадра при загрузке/выгрузке чанков.\n\nТребуется OpenGL 4.4 или ARB_buffer_storage.",
+  "sodium.options.chunk_memory_allocator.name": "Распределитель памяти",
+  "sodium.options.chunk_memory_allocator.tooltip": "Выбирает распределитель памяти, используемый для рендеринга чанков.\n- ASYNC: Быстрейший вариант, хорошо работает с большинством современных графических драйверов.\n- SWAP: Запасной вариант для старых графических драйверов. Может существенно увеличить использование памяти.",
+  "sodium.options.chunk_memory_allocator.async": "Async",
+  "sodium.options.chunk_memory_allocator.swap": "Swap",
+  "sodium.options.chunk_update_threads.name": "Потоки обновления чанков",
+  "sodium.options.chunk_update_threads.tooltip": "Задаёт количество потоков для сборки чанков. Использование большего количества потоков способствует повышению скорости обновления и загрузки чанков, но может негативно повлиять на время кадра.",
+  "sodium.options.always_defer_chunk_updates.name": "Всегда откладывать обновления",
+  "sodium.options.always_defer_chunk_updates.tooltip": "Если включено, рендеринг никогда не будет ожидать завершения обновлений чанков, даже если они важны. Это способствует сильному увеличению частоты кадров в некоторых ситуациях, но может привести к появлению существенных визуальных ошибок в мире.",
+  "sodium.options.buttons.undo": "Отмена",
+  "sodium.options.buttons.apply": "Применить",
+  "sodium.options.buttons.donate": "Угостите нас кофе!"
```

Translations for ModMenu summary and description (summary is a shortened version of the description that can fit into a limited space)
```suggestion
  "sodium.options.buttons.donate": "Угостите нас кофе!",
   "modmenu.summaryTranslation.sodium": "Свободный и открытый мод для оптимизации Minecraft.",
   "modmenu.descriptionTranslation.sodium": "Свободный мод для оптимизации Minecraft с открытым исходным кодом, призванный уменьшить зависания и улучшить частоту кадров."
```